### ISSUE TITLE
A11y_AzureDataStudio_Connect to a SQL Server_Focus Visible: Focus not visible for connection icon while using keyboard

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#18587]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG2.4.7; #Win32; #AINotInScope; #Focus Visible; 

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Verify the focus visible for connection icon or not by using keyboard

### Note:
Similar issue occurred for Notebook icon 
 
### Actual Result:
Focus not visible for connection icon while using keyboard

### Expected Result:
Focus should visible for connection icon while using keyboard

### Attachment:

![A11y_AzureDataStudio_Connect to a SQL Server_Focus Visible](https://user-images.githubusercontent.com/100189407/156111808-4431b9e4-61e1-4eaf-854b-697c5439a80d.gif)

**Comment by VijayaBharathyA** [comment]
Verified in the latest Environment
Azure Data studio Version: 1.37.0
Issue is Fixed.
![18587_Fixed](https://user-images.githubusercontent.com/100189407/178216939-66f1fc29-d354-45b5-bb62-3080a511b47b.gif)
